.gitlab-ci.yml

```diff
@@ -24,6 +24,8 @@ docker_build:
     - docker
   script:
     - DOCKER_HOST=tcp://localhost:2375 docker login -u $CI_REGISTRY_USER -p $CI_JOB_TOKEN $CI_REGISTRY
+    # Prune images older than 2 days
+    - docker system prune --filter "until=48h" -f -a
```

Comment:
2 days seems pretty aggressive given that I don't think we were pruning at all before and didn't run into disk space issues for ~weeks? (I might be mistaken about this) Would it make sense to bump this up a bit to reduce the number of unnecessary downloads?

Reply:
This is all dependent on how often pip/conda requirements change and if the base image changes. Could increase it to something like a week, but not sure that this wouldn't bite us in a week like these where we have lots of changes in our images. We could also increase the disk size as right now there is only about 80gb free at any time.